articles/models.py

```diff
@@ -46,3 +47,31 @@ def __str__(self) -> str:
 def slug_pre_save(sender: Any, instance: Any, **kwargs: Any) -> None:
     if instance.slug is None or instance.slug == "":
         instance.slug = slugify(f"{instance.title}-{instance.lookup_id}")
+
+
+class Comment(models.Model):
+    """
+    Handles CRUD on a comment that has been made on article
+    """
+
+    lookup_id = models.UUIDField(
+        default=uuid.uuid4, editable=False, unique=True, max_length=255
+    )
+    body = models.TextField(max_length=500, blank=False, null=False)
+    createdAt = models.DateTimeField(auto_now_add=True)
```

Comment:
Use the timestamp file for created_at and updated_at
Also check your variable naming, refer to PEP8 standards